spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/TaskConfigurationProperties.java

```diff
@@ -146,4 +155,20 @@ public boolean isUseKubernetesSecretsForDbCredentials() {
 	public void setUseKubernetesSecretsForDbCredentials(boolean useKubernetesSecretsForDbCredentials) {
 		this.useKubernetesSecretsForDbCredentials = useKubernetesSecretsForDbCredentials;
 	}
+
+	private void logDeprecationWarning() {
+		logDeprecationWarning(null);
+	}
+
+	private void logDeprecationWarning(String newMethodName) {
+		String callingMethodName = Thread.currentThread().getStackTrace()[2].getMethodName();
+		LOGGER.warn(this.getClass().getName() + "." + callingMethodName
+				+ " is deprecated. Please use " + ComposedTaskRunnerConfigurationProperties.class.getName() + "."
+				+ (newMethodName != null ? newMethodName : callingMethodName));
+	}
+
+	void setComposedTaskRunnerConfigurationProperties(ComposedTaskRunnerConfigurationProperties
+															  composedTaskRunnerConfigurationProperties) {
```

Comment:
May want to reformat this code.    Tabbing is not lining up.   Your friendly neighborhood nitpick.